test/integration/taxon_browsing_test.rb

```diff
@@ -283,6 +285,20 @@ def and_i_can_see_the_sub_topics_grid
     end
   end
 
+  def and_i_can_see_the_in_page_nav
```

Comment:
I don't think this test actually tests the presence of the in-page nav, it seems to check the new design of the sub-topic list. It makes sense to still keep this as something we want to check, but you might want to write an additional separate check for seeing the in-page nav. Things we may want to check there:

- Presence of the contents list component
- Presence of links in the contents list component, e.g: links to the sub-topic list

Reply:
👍